server/db.js

```diff
@@ -1,9 +1,15 @@
 const mongoose = require("mongoose");
 
-const connectDB = async () => {
-  const conn = await mongoose.connect(process.env.MONGO_URI);
+const connectDB = async() => {
+    let db_uri
```

Comment:
`dbUri` let's preference camelCase in JavaScript as it's the convention. Thanks!